src/main/views/govukTemplate.njk

```diff
@@ -31,100 +30,101 @@
   {% endif %}
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta property="og:image" content="{{ asset_paths['images_vendor'] }}/opengraph-image.png?{{ govuk_template_version }}">
 
   {% block head %}{% endblock %}
+
+  <meta property="og:image" content="{{ asset_paths['images_vendor'] }}/opengraph-image.png?{{ govuk_template_version }}">
 </head>
 
 <body class="{% block body_classes %}{% endblock %}">
+  <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
```

Comment:
@geek-soft won't this get blocked by CSP? This is why it was removed as the js we were using wasn't using it. Can also be added if it has a hash of the js